README.md

```diff
@@ -4,15 +4,35 @@
 * Dates are represented as tuple terms (D,M,Y).
 * Library converts absolute dates into relative dates.
 * Library finds out the weekday of a date.
+* Library can create date tuples for every day in a given month or year.
 * Day differences are realized using Julian Day Numbers, see http://quasar.as.utexas.edu/BillInfo/JulianDatesG.html .
 * Week days are realized using a formula from https://de.wikipedia.org/wiki/Wochentagsberechnung
 
 
 
 # Input Predicates
 
+## date_consider_month/2( M, Y )
+
+Creates all dates of the given month in the given year that need to be considered.
+
+|Argument | Domain                     | Note          |
+|:--------|:---------------------------|:--------------|
+| M       | number (1-12)              | Month
+| Y       | number (1XXX-2XXX)         | Year
+
+## date_consider_year/2( Y )
+
+Creates all dates of the given year that need to be considered.
+
+|Argument | Domain                     | Note          |
+|:--------|:---------------------------|:--------------|
+| Y       | number (1XXX-2XXX)         | Year
+
 ## date_consider/1( (D,M,Y) )
 
+It is recommended to use date_consider_month/2 or date_consider_year/1, because it prevents the accidental creation of 'illegal' dates i.e. 29th of February in a non leap year.
```

Comment:
Maybe better.
:warning: Do not use!  It allows the creation of  *illegal* dates i.e. 29th of February in a non leap year.
It is recommended to use date_consider_month/2 or date_consider_year/1

Reply:
@MaxOstrowski  Maybe we should remove date_consider/1 from the README at all. If it should not be used as Input predicate. I wish we could make predicates private.

Reply:
Maybe add integrity constraints for *ilegal* dates? or a save alternative like `checked_date_consider`?